.vib/kubectl/goss/vars.yaml

```diff
@@ -0,0 +1,11 @@
+binaries:
+  - kubectl
+  - jq
```

Comment:
jq is installed via system packages, not sure if it is worth checking it here. WDYT?

Reply:
We don't usually check system pkgs installed via Dockerfile with the exception of ca-certificates, which enables https. The reason for this is that testing Dockerfile logic means we would be checking one of the following:

1. Base image distro
2. Docker logic

Both of them are external to the asset itself and are expected to work as expected and thus should not require explicit testing in our side.

That said, this is the current approach but please share your thoughts on this.

Reply:
Thank you! I didn't have strong thoughts on this. The current approach makes sense to me 👍